src/commands.cpp

```diff
@@ -5129,6 +5129,52 @@ void CommandRegisterPushNotification::procresult()
         client->app->registerpushnotification_result(API_EINTERNAL);
     }
 }
+
+CommandArchiveChat::CommandArchiveChat(MegaClient *client, handle chatid, bool archive)
+{
+    this->client = client;
```

Comment:
No need to assign the client here, since it's already set at the parent class `Command`